jobs/JGDAS_ATMOS_ANALYSIS_DIAG

```diff
@@ -1,49 +1,7 @@
 #! /usr/bin/env bash
 
 source "${HOMEgfs}/ush/preamble.sh"
-
-##############################################
-# make temp directory
-##############################################
-export DATA=${DATA:-${DATAROOT}/${jobid}}
-mkdir -p ${DATA}
-cd ${DATA}
-
-
-##############################################
-# Run setpdy and initialize PDY variables
-##############################################
-export cycle="t${cyc}z"
-setpdy.sh
-. ./PDY
-
-
-##############################################
-# Determine Job Output Name on System
-##############################################
-export pid=${pid:-$$}
-export pgmout="OUTPUT.${pid}"
-export pgmerr=errfile
-
-
-#############################
-# Source relevant config files
-#############################
-export EXPDIR=${EXPDIR:-${HOMEgfs}/parm/config}
-configs="base anal analdiag"
-for config in ${configs}; do
-    . ${EXPDIR}/config.${config}
-    status=$?
-    [[ ${status} -ne 0 ]] && exit ${status}
-done
-
-
-##########################################
-# Source machine runtime environment
-##########################################
-. ${HOMEgfs}/env/${machine}.env anal
```

Comment:
This job is passing `anal` as the argument for sourcing `$machine.env` file.  In the `jjob_header.sh`, it is going to use `analdiag` as that is set in the rocoto job `analdiag.sh`. There is no entry for `analdiag` in `$machine.env`, atleast not in HERA.env

Reply:
Not in `ORION.env` either. Must not be important, since it completes and produces bit-identical results.

Reply:
(That's not a reason not to add an entry for analdiag, just maybe it should be empty?)

Reply:
Refactored the jjob header to take in the job name to pass to the environment script.